operator/v1/types_network.go

```diff
@@ -80,6 +80,11 @@ type NetworkSpec struct {
 	// If not specified, sensible defaults will be chosen by OpenShift directly.
 	// Not consumed by all network providers - currently only openshift-sdn.
 	KubeProxyConfig *ProxyConfig `json:"kubeProxyConfig,omitempty"`
+
+	// exportNetworkFlows enables and configures the export of network flows from OVS
+	// by using protocols netFlow, sFlow or ipfix.
+	// +optional
+	ExportNetworkFlows *ExportNetworkFlows `json:"exportNetworkFlows,omitempty"`
```

Comment:
why would I set this.  How would I know the values?

Reply:
This is for the use case of customers willing to collect and analyze flows from OVN-Kubernetes.
If you want more context, I linked an enhancement to this PR:

https://github.com/openshift/enhancements/pull/581

Reply:
> This is for the use case of customers willing to collect and analyze flows from OVN-Kubernetes.
> If you want more context, I linked an enhancement to this PR:

Can you provide a two sentence summary for the godoc?

Reply:
This is basically a fancy complicated networking-specific form of logging. This configuration object is the equivalent of "give the hostname and port of your remote syslog server". If the user already has a flow collector set up, then they will know what needs to be put here, and if they don't, then they have more work ahead of them than we can explain in the API docs.